src/mujinzmq.cpp

```diff
@@ -495,7 +494,6 @@ void ZmqServer::_InitializeSocket(boost::shared_ptr<zmq::context_t> context)
 void ZmqServer::_DestroySocket()
 {
     if (!!_socket) {
-        _socket->setsockopt(ZMQ_LINGER, 0);
```

Comment:
@felixvd oh, should this be still useful for binpickingtask?

Reply:
Whoops. Removed this.